docs/GettingStarted.md

```diff
@@ -66,7 +69,8 @@ $ cmake -G Ninja ../llvm \
     -DLLVM_ENABLE_PROJECTS="mlir" \
     -DLLVM_TARGETS_TO_BUILD="X86;RISCV" \
```

Comment:
Nit: this should probably just be "host".

Reply:
not sure what you mean by this

Reply:
`s/X86;RISCV/host/`